install_config/web_console_customization.adoc

```diff
@@ -537,20 +537,6 @@ how to configure HAProxy routers to allow wildcard routes].
 Sometimes features are available in Technology Preview. By default, these
 features are disabled in the web console and hidden from end users.
 
-Web console features currently in Technology Preview include:
```

Comment:
@jwforres Looks like there are 2 features for 3.4 that are listed in Technology Preview according to [3.4 release notes](https://docs.openshift.com/container-platform/3.4/release_notes/ocp_3_4_release_notes.html#ocp-34-kubernetes-deployments-support). Should those be listed here?